src/latexify/output.py

```diff
@@ -0,0 +1,120 @@
+"""Output of the frontend function decorators."""
+
+from __future__ import annotations
+
+import abc
+from typing import Any, Callable
+
+from latexify import exceptions, frontend
```

Comment:
Circular import.

```suggestion
from latexify import exceptions
```

Reply:
Should I import locally in the method instead? I think we still need `frontend` because of `get_latex`.

Reply:
Circular imports can be usually resolved by separating files. In this case the problem is caused by coexisting `get_latex` and wrapper functions, so it should be resolved by splitting `get_latex` out to another file.

Reply:
moved into `codegen/generate_latex.py`